### PR TITLE
9497 identify consolidated cases in qc display

### PR DESCRIPTION
**Pull request by rgoshen**
This PR addressed PO feedback concerning when uploading a PDF file DocumentQC Processed did not show either Lead Case icon or consolidated group icon.

### CODE REVIEW

File: shared/admin-tools/user/admin.js [hunk 1]
```diff
@@ -76,6 +76,56 @@ const deactivateAdminAccount = async () => {
     .promise();
 };
 
+/**
+ * This verifies that the USTC admin user is disabled in Cognito
+ */
+const verifyAdminUserDisabled = async ({ attempt }) => {
+  const cognito = new CognitoIdentityServiceProvider({ region: 'us-east-1' });
```
Comment (ujahio) [inline shared/admin-tools/user/admin.js:83]: nit-pick: Possible use a constant for `us-east-1`

File: shared/admin-tools/user/verify-ustc-admin-user-disabled.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const { verifyAdminUserDisabled } = require('./admin');
+
+(async () => {
+  console.log('== Verifying USTC admin user is disabled');
```
Comment (ujahio) [inline shared/admin-tools/user/verify-ustc-admin-user-disabled.js:4]: Question/nit-pick: Depending on where the logs are presented, does `ApplicationContext.logger.info('== Verifying USTC admin user is disabled`)` work better?

File: web-client/integration-tests/petitionsClerkBlockConsolidatedCasesJourney.test.js [hunk 3]
```diff
@@ -46,16 +48,16 @@ describe('Manually block consolidated cases', () => {
   loginAs(cerebralTest, 'docketclerk@example.com');
```
Comment (ujahio) [inline web-client/integration-tests/petitionsClerkBlockConsolidatedCasesJourney.test.js]: Delete?

**Review by ujahio — APPROVED** [review]

File: shared/admin-tools/user/admin.js (hunk 1 above)
Comment (rgoshen) [reply shared/admin-tools/user/admin.js:83]: Unfortunately, I think this is an artifact with github.  I did not make this change.

File: shared/admin-tools/user/verify-ustc-admin-user-disabled.js (hunk 2 above)
Comment (rgoshen) [reply shared/admin-tools/user/verify-ustc-admin-user-disabled.js:4]: Unfortunately, I think this is an artifact with github. I did not make this change.

File: web-client/integration-tests/petitionsClerkBlockConsolidatedCasesJourney.test.js (hunk 3 above)
Comment (rgoshen) [reply web-client/integration-tests/petitionsClerkBlockConsolidatedCasesJourney.test.js]: good catch

**Pull request merged**